src/points/PointsDiscoverCard.test.tsx

```diff
@@ -0,0 +1,81 @@
+import { fireEvent, render } from '@testing-library/react-native'
+import React from 'react'
+import { Provider } from 'react-redux'
+import { PointsEvents } from 'src/analytics/Events'
+import ValoraAnalytics from 'src/analytics/ValoraAnalytics'
+import { navigate } from 'src/navigator/NavigationService'
+import { Screens } from 'src/navigator/Screens'
+import PointsDiscoverCard from 'src/points/PointsDiscoverCard'
+import { RootState } from 'src/redux/store'
+import { getFeatureGate } from 'src/statsig/index'
+import { RecursivePartial, createMockStore } from 'test/utils'
+
+jest.mock('src/analytics/ValoraAnalytics')
+jest.mock('src/statsig')
+
+const renderPointsDiscoverCard = (storeOverrides?: RecursivePartial<RootState>) => {
+  const store = createMockStore(storeOverrides)
+
+  const tree = render(
+    <Provider store={store}>
+      <PointsDiscoverCard />
+    </Provider>
+  )
+
+  return {
+    store,
+    ...tree,
+  }
+}
+
+describe('PointsDiscoverCard', () => {
+  beforeEach(() => {
+    jest.clearAllMocks()
+  })
+
+  it('renders when feature gate is enabled', () => {
+    jest.mocked(getFeatureGate).mockReturnValue(true)
```

Comment:
nit: maybe this should be in the beforeEach as a default for all tests so you only need to mock it to return false for 1 test case?